mobile_back_qti/cpp/backend_qti/mlperf_helper.h

```diff
@@ -31,7 +31,7 @@ static void process_config(const mlperf_backend_configuration_t *configs,
   backend_data->loadOffTime_ = 2;
   backend_data->loadOnTime_ = 100;
   backend_data->useIonBuffers_ = true;
-  backend_data->acceleratorName_=configs->accelerator_desc;
+  backend_data->acceleratorName_=configs->accelerator;
```

Comment:
@mohitmundhragithub I replace `accelerator_desc` with `accelerator` for convenient. I assume the QTI backend should work normally since the `accelerator_desc` string was just used to report back its name in `mlperf_backend_accelerator_name`. Please check it, thanks.

Reply:
mlperf_backend_accelerator_name -- Is this API not used by frontend? if not then its fine for Qualcomm as well.

Reply:
It's used only for showing the accelerator name in the settings screen.